### ISSUE TITLE
Error during db:seed_fu script

### ISSUE THREAD

**Issue by ptrikutam** [code-corps/deprecated-code-corps-rails-api#213]
Looks like the `posts.rb` fixture is executed before the `users.rb` fixture populated, causing the error below.

```
$ bundle exec rake db:seed_fu

== Seed from /Users/pavan/Development/code-corps-api/db/fixtures/organizations.rb
 - Organization {:id=>1, :name=>"Code Corps", :slug=>"code_corps"}

== Seed from /Users/pavan/Development/code-corps-api/db/fixtures/posts.rb
 - Post {:id=>0, :title=>"A task", :body=>"This is a basic task", :markdown=>"This is a basic task", :project_id=>1, :user_id=>2, :post_type=>"task"}
rake aborted!
ActiveRecord::InvalidForeignKey: PG::ForeignKeyViolation: ERROR:  insert or update on table "posts" violates foreign key constraint "fk_rails_5b5ddfd518"
DETAIL:  Key (user_id)=(2) is not present in table "users".
: INSERT INTO "posts" ("aasm_state", "id", "title", "body", "markdown", "project_id", "user_id", "created_at", "updated_at") VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9) RETURNING "id"
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:46:in `eval'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:602:in `block in exec_cache'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract_adapter.rb:472:in `block in log'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract_adapter.rb:466:in `log'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:601:in `exec_cache'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:585:in `execute_and_clear'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql/database_statements.rb:160:in `exec_query'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql/database_statements.rb:192:in `exec_insert'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:108:in `insert'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/query_cache.rb:14:in `insert'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/relation.rb:64:in `insert'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/persistence.rb:524:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/counter_cache.rb:139:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/locking/optimistic.rb:75:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/attribute_methods/dirty.rb:132:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/callbacks.rb:306:in `block in _create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:117:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:117:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:555:in `block (2 levels) in compile'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:505:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:505:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:92:in `__run_callbacks__'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:778:in `_run_create_callbacks'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/callbacks.rb:306:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/timestamp.rb:57:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/persistence.rb:504:in `create_or_update'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/callbacks.rb:302:in `block in create_or_update'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:117:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:117:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:555:in `block (2 levels) in compile'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:505:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:505:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:92:in `__run_callbacks__'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:778:in `_run_save_callbacks'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/callbacks.rb:302:in `create_or_update'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/persistence.rb:120:in `save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/bullet-5.0.0/lib/bullet/active_record42.rb:27:in `save_with_bullet'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/validations.rb:37:in `save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/attribute_methods/dirty.rb:21:in `save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:286:in `block (2 levels) in save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:351:in `block in with_transaction_returning_status'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:211:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:220:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:348:in `with_transaction_returning_status'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:286:in `block in save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:301:in `rollback_active_record_state!'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:285:in `save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:74:in `seed_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:36:in `block (2 levels) in seed'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:36:in `map'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:36:in `block in seed'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:211:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:220:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:35:in `seed'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/active_record_extension.rb:32:in `seed'
(eval):2:in `block (3 levels) in run_file'
(eval):1:in `times'
(eval):1:in `block (2 levels) in run_file'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:46:in `eval'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:46:in `block (2 levels) in run_file'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:58:in `block in open'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:57:in `open'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:57:in `open'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:36:in `block in run_file'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:213:in `block in transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/transaction.rb:184:in `within_new_transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:213:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:220:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:35:in `run_file'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:26:in `block in run'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:25:in `each'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:25:in `run'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu.rb:29:in `seed'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/tasks/seed_fu.rake:36:in `block (2 levels) in <top (required)>'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/bin/ruby_executable_hooks:15:in `eval'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/bin/ruby_executable_hooks:15:in `<main>'
PG::ForeignKeyViolation: ERROR:  insert or update on table "posts" violates foreign key constraint "fk_rails_5b5ddfd518"
DETAIL:  Key (user_id)=(2) is not present in table "users".
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:46:in `eval'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:602:in `block in exec_cache'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract_adapter.rb:472:in `block in log'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract_adapter.rb:466:in `log'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:601:in `exec_cache'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:585:in `execute_and_clear'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql/database_statements.rb:160:in `exec_query'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql/database_statements.rb:192:in `exec_insert'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:108:in `insert'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/query_cache.rb:14:in `insert'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/relation.rb:64:in `insert'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/persistence.rb:524:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/counter_cache.rb:139:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/locking/optimistic.rb:75:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/attribute_methods/dirty.rb:132:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/callbacks.rb:306:in `block in _create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:117:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:117:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:555:in `block (2 levels) in compile'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:505:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:505:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:92:in `__run_callbacks__'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:778:in `_run_create_callbacks'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/callbacks.rb:306:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/timestamp.rb:57:in `_create_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/persistence.rb:504:in `create_or_update'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/callbacks.rb:302:in `block in create_or_update'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:117:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:117:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:555:in `block (2 levels) in compile'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:505:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:505:in `call'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:92:in `__run_callbacks__'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activesupport-4.2.5/lib/active_support/callbacks.rb:778:in `_run_save_callbacks'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/callbacks.rb:302:in `create_or_update'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/persistence.rb:120:in `save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/bullet-5.0.0/lib/bullet/active_record42.rb:27:in `save_with_bullet'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/validations.rb:37:in `save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/attribute_methods/dirty.rb:21:in `save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:286:in `block (2 levels) in save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:351:in `block in with_transaction_returning_status'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:211:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:220:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:348:in `with_transaction_returning_status'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:286:in `block in save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:301:in `rollback_active_record_state!'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:285:in `save'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:74:in `seed_record'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:36:in `block (2 levels) in seed'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:36:in `map'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:36:in `block in seed'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:211:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:220:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/seeder.rb:35:in `seed'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/active_record_extension.rb:32:in `seed'
(eval):2:in `block (3 levels) in run_file'
(eval):1:in `times'
(eval):1:in `block (2 levels) in run_file'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:46:in `eval'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:46:in `block (2 levels) in run_file'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:58:in `block in open'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:57:in `open'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:57:in `open'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:36:in `block in run_file'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:213:in `block in transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/transaction.rb:184:in `within_new_transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:213:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/activerecord-4.2.5/lib/active_record/transactions.rb:220:in `transaction'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:35:in `run_file'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:26:in `block in run'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:25:in `each'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu/runner.rb:25:in `run'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/seed-fu.rb:29:in `seed'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/gems/seed-fu-2.3.5/lib/tasks/seed_fu.rake:36:in `block (2 levels) in <top (required)>'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/bin/ruby_executable_hooks:15:in `eval'
/Users/pavan/.rvm/gems/ruby-2.2.3@codecorps/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => db:seed_fu
(See full trace by running task with --trace)
```

**Comment by joshsmith** [comment]
Ah, I closed #198 which had a fix for this incorporated, but didn't make it in for unrelated reasons.

My fix is to have the following name changes:
- db/fixtures/users.rb → db/fixtures/001_users.rb
- db/fixtures/organizations.rb → db/fixtures/002_organizations.rb
- db/fixtures/projects.rb → db/fixtures/003_projects.rb
- db/fixtures/posts.rb → db/fixtures/004_posts.rb

There's a further bug in `posts.rb` that starts numbering from 0, so that needs fixed.

And we should also be setting `post.aasm_state = "published"` in these fixtures, as well.

**Comment by joshsmith** [comment]
I'd be happy to take a PR for this, if you're interested. Would be a very quick one.

**Comment by ptrikutam** [comment]
Fixed in #214.
